includes/wikia/GlobalTitle.php

```diff
@@ -70,6 +70,12 @@ public static function newFromText( $text, $namespace = NS_MAIN, $city_id = null
 		}
 
 		$filteredText = Sanitizer::decodeCharReferences( $text );
+
+		if ( $namespace === NS_SPECIAL ) {
+			$localName = SpecialPageFactory::getLocalNameFor( $text );
```

Comment:
Addressed in https://github.com/Wikia/app/pull/11379/commits/f8d94a4b348a59e4c0de532c2c78ed99356c4a49